x-pack/plugins/cloud_security_posture/public/components/cloud_security_data_table/fields_selector/fields_selector_modal.tsx

```diff
@@ -47,18 +45,6 @@ export const FieldsSelectorModal = ({
   onRemoveColumn,
   onResetColumns,
 }: FieldsSelectorModalProps) => {
-  const [isFilterSelectedEnabled, setIsFilterSelectedEnabled] = useSessionStorage(
```

Comment:
I moved this to the `field_selector_table` mainly because I removed the `useEffect` in the  field_selector_table, but also this caused the table to re-render 3 times because the change happened in the parent modal container.